website/integrations/services/wazuh/index.md

```diff
@@ -0,0 +1,224 @@
+![](./property-mapping3.png)
+
+### Step 4 - saml application
+
+Now create an application to use the newly created provider. `Applications` -> `Applications` -> `Create`
+
+`Name: wazuh-saml`
+`Slug: wazuh-saml`
+`Provider: SAML`
+`Policy Engine: any`
+
+![](./application.png)
+
+You can change the UI / upload a logo so that in the applications overview you have a nice layout and can easily identify the new wazuh app.
+![](./applications_overview.png)
+
+### Step 5 - metadata + wazuh opensearch-security configuration
+
+Now download the metadata file `saml_authentik_meta.xml` from the `Applications` -> `Provider` -> `Related Objects` -> `Download`
+
+and copy/save it on the wazuh server - ideally under `/etc/wazuh-indexer/opensearch-security/idp-metadata.xml`
+![Alt text](./metadata.png)
+
+Next up change the `/etc/wazuh-indexer/opensearch-security/config.yml` and make sure it looks like the one below
+
+-   you need to adjust the `metadata_file` if your name differs from the one shown above/below
+-   `entity_id` needs to change twice, once in the `idp` section and once in the `sp` section - you can look it up in the metadata xml file - search for `entityID`
+    ![Alt text](entityid.png)
+-   adapt the `kibana_url` to match your wazuh dashboard url - e.g. `https://wazuh.myhomelab.com/`
+-   copy/paste the `exchange_key`, you can get it from the metadata file (find the key between the `<ds:X509Certificate></ds:X509Certificate>` tags, it usually starts with MII...) - DO NOT FORGET TO PUT QUOTES AROUND THE CERTIFICATE
+-   make sure to adjust the ownership and access rights via the following commands:
+
+```bash
+sudo chown wazuh-indexer:wazuh-indexer /etc/wazuh-indexer/opensearch-security/idp-metadata.xml
+
+sudo chmod 640 /etc/wazuh-indexer/opensearch-security/idp-metadata.xml
+```
+
+```yml
+authc:
+  basic_internal_auth_domain:
+    description: "Authenticate via HTTP Basic + SAML against internal users database"
+    http_enabled: true
+    transport_enabled: true b
+    order: 0
+    http_authenticator:
+      type: basic
+      challenge: false
+    authentication_backend:
+      type: intern
+  saml_auth_domain:
+    http_enabled: true
+    transport_enabled: false
+    order: 1
+    http_authenticator:
+      type: saml
+      challenge: true
+      config:
+        idp:
+          metadata_file: "/etc/wazuh-indexer/opensearch-security/idp-metadata.xml"
+          entity_id: "wazuh-saml"
+        sp:
+          entity_id: "wazuh-saml"
+        kibana_url: "https://<YOUR_WAZUH_IP_OR_HOSTNAME>"
+        roles_key: Roles
+        exchange_key: "MIIGIODCCA+ygAwIBAgITREIpdNn++usuy4Gq5g0W2/SQs..."
+     authentication_backend:
+       type: noop
+```
+
+### Step 6 - securityadmin.sh
+
+After you save the `config.yml` you need to restart the wazuh manager and tell it to load the modified file - this can be done automagically using the `securityadmin.sh` wazuh provides with the following command:
```

Comment:
```suggestion
After you save the `config.yml` you need to restart the wazuh manager and tell it to load the modified file - this can be done automagically using the `securityadmin.sh` file that wazuh provides with the following command:
```